Curs/src/events/evaluationEvents.js

```diff
@@ -0,0 +1,18 @@
+interviewApp.evaluationEvents = function() {
+
+    const goToTheNewEvaluationPage = function() {
+        interviewApp.newEvaluationModule.init() // show new-evaluation page
+    }
+   
+    const newEvaluationButton = document.getElementById("new-evaluation-page") // get newEvaluationButton by his ID
+
+    newEvaluationButton.addEventListener("click", goToTheNewEvaluationPage) // add click event listener on newEvaluationButton
+
+    const addDetailsButtons = document.getElementsByClassName("add-button");
+    function bindEvent(el) {
+        el.addEventListener("click", goToTheNewEvaluationPage);
+    }
+    Array.prototype.forEach.call(addDetailsButtons, bindEvent);
```

Comment:
Can you explain what this does?

Reply:
On the evaluation page exists a nice blue button :)
I added an eventListener on it, and when I click there to take me to the New-Evaluation page and also to be able to enter a new entry with Candidate, Interviewer and Date.